src/brainCloudClient-file.js

```diff
@@ -68,16 +68,18 @@ function BCFile() {
 	 * @param xhr The XMLHttpRequest object that the brainCloud client will
 	 * use to upload the file.
 	 * @param file The file object
-	 * @param uploadId The upload id obtained via prepareFileUpload()
+	 * @param uploadId The upload id obtained via prepareUserUpload()
+     * @param peerCode - optional - peerCode.  A Peer needs to allow prepareUserUpload 
 	 */
-	bc.file.uploadFile = function(xhr, file, uploadId) {
+	bc.file.uploadFile = function(xhr, file, uploadId, peerCode = "") {
```

Comment:
please confirm if i need to use 

bc.file.uploadFile = function(xhr, file, uploadId, peerCode) {
    // use this if you specifically want to know if peerCode was passed
    if (peerCode === undefined) {
        // peerCode was not passed
        peerCode = "";
    }
    // use this if you know that a truthy value comparison will be enough
    if (peerCode) {
        // peerCode was passed and has truthy value
    } else {
        // peerCode was not passed or has falsy value
    }
    // use this to set b to a default value (using truthy comparison)
    peerCode = peerCode || "";

instead of the above

Reply:
ES5 please yea